### PR TITLE
Add extra dictionaries.

### PR DESCRIPTION
**Pull request by Tas-sos**
* [X] Added extra dictionaries.

Example
========

```
ansible-playbook -u debian -i rciam-deploy-inv/eosc-portal-demo/hosts.ini saml2samlservers.yml --vault-password-file=../vaultRCIAM --tags "ssp_modules:dict"
```

```
TASK [Gathering Facts] ********************************************************************************************************************************************************************************************
ok: [example.com]

TASK [ssp-modules : Include OS-specific variables] ****************************************************************************************************************************************************************
ok: [example.com] => (item=/..../rciam-deploy/roles/ssp-modules/vars/Debian.yml)

TASK [ssp-modules : Add extra dictionaries] ***********************************************************************************************************************************************************************
ok: [example.com] => (item={'name': 'discopower', 'src_path': '/..../rciam-deploy/rciam-deploy-inv/eosc-portal-demo/files/templates/dictionaries/tabs.definition.json.j2', 'dest_path': '/srv/simplesamlphp/proxy/modules/discopower/dictionaries/tabs.definition.json'})

changed: [example.com] => (item={'name': 'themevanilla_default', 'src_path': '/..../rciam-deploy/rciam-deploy-inv/eosc-portal-demo/files/templates/dictionaries/themevanilla-release-3.0.0/default.definition.json', 'dest_path': '/srv/simplesamlphp/proxy/modules/themeeosc/dictionaries/tabs.definition.json'})

changed: [example.com] => (item={'name': 'themevanilla_discopower', 'src_path': '/..../rciam-deploy/rciam-deploy-inv/eosc-portal-demo/files/templates/dictionaries/themevanilla-release-3.0.0/discopower.definition.json', 'dest_path': '/srv/simplesamlphp/proxy/modules/themeeosc/dictionaries/tabs.definition.json'})

ok: [example.com] => (item={'name': 'themevanilla_policy', 'src_path': '/..../rciam-deploy/rciam-deploy-inv/eosc-portal-demo/files/templates/dictionaries/themevanilla-release-3.0.0/policy.definition.json', 'dest_path': '/srv/simplesamlphp/proxy/modules/themeeosc/dictionaries/policy.definition.json'})

ok: [example.com] => (item={'name': 'themevanilla_userid_error', 'src_path': '/..../rciam-deploy/rciam-deploy-inv/eosc-portal-demo/files/templates/dictionaries/themevanilla-release-3.0.0/userid_error.definition.json', 'dest_path': '/srv/simplesamlphp/proxy/modules/themeeosc/dictionaries/userid_error.definition.json'})

PLAY RECAP ********************************************************************************************************************************************************************************************************
example.com : ok=3    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   
```

:link: https://github.com/grnet/rciam-deploy-inv/pull/64

### CODE REVIEW

**Review by nikosev — COMMENTED** [review]
The task seems ok to me, but I'm not sure about the tags
There is this open PR: https://github.com/rciam/rciam-deploy/pull/42/commits/9f0a96ae52d3500a05f0dd1311af67fa61d17eb8
in which there have been added some tags to the task files.
So, maybe these tags should be removed

**Comment by Tas-sos** [comment]
Thanks Nick,

Your comment about the **tags** is right and interesting. Although I did not want to deal with it now, I noticed that there were no tags at all.

I just pushed a few more changes, now about the tags. I added the tags according with the [ssp](https://github.com/rciam/rciam-deploy/tree/devel/roles/ssp/tasks) role so that it has consistency/similarity with each other.
_By the way, if you think that we are going beyond the scope of this pull request, of course I can remove all the tags and deal with them in your pull request._

File: roles/ssp-modules/tasks/configure-common.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@
   become: yes
   with_items: "{{ ssp_mods_extra_enabled }}"
   when: item.src_config_path is defined and item.dest_config_path is defined
+  tags:
+    - ssp_modules:config:config
```
Comment (NicolasLiampotis) [inline roles/ssp-modules/tasks/configure-common.yml:14]: Remove tag

File: roles/ssp-modules/tasks/configure-common.yml [hunk 2]
```diff
@@ -31,4 +47,5 @@
     msg:
       - The role also installs Themes, but doesn't produce the CSS files.
       - To produce them, you must install `sass` and follow the installation instructions for each theme.
-
+  tags:
+    - ssp_modules:config:config
```
Comment (NicolasLiampotis) [inline roles/ssp-modules/tasks/configure-common.yml:51]: Remove tag

File: roles/ssp-modules/tasks/install-common.yml [hunk 3]
```diff
@@ -6,6 +6,8 @@
     name: unzip
     state: latest
   become: yes
+  tags:
+    - ssp_modules:install:install
```
Comment (NicolasLiampotis) [inline roles/ssp-modules/tasks/install-common.yml:10]: Remove tag

File: roles/ssp-modules/tasks/install-common.yml [hunk 4]
```diff
@@ -19,6 +21,8 @@
   become: yes
   with_items: "{{ ssp_mods_extra_enabled }}"
   when: item.repo_url is defined
+  tags:
+    - ssp_modules:install:install
```
Comment (NicolasLiampotis) [inline roles/ssp-modules/tasks/install-common.yml:25]: Remove tag

File: roles/ssp-modules/tasks/install-common.yml [hunk 5]
```diff
@@ -31,13 +35,17 @@
   with_items: "{{ ssp_mods_extra_enabled }}"
   when: item.zip_url is defined
   register: zip_downloaded
+  tags:
+    - ssp_modules:install:install
```
Comment (NicolasLiampotis) [inline roles/ssp-modules/tasks/install-common.yml:39]: Remove tag

File: roles/ssp-modules/tasks/install-common.yml [hunk 6]
```diff
@@ -31,13 +35,17 @@
...
 
 
 - name: Move unarchived module files from /tmp/ to module directory (zip only) [1/3]
   command: "mv /tmp/{{ item.fullname }} {{ item.path }}"
   become: yes
   with_items: "{{ ssp_mods_extra_enabled }}"
   when: zip_downloaded.changed and item.zip_url is defined and item.fullname is defined
+  tags:
+    - ssp_modules:install:install
```
Comment (NicolasLiampotis) [inline roles/ssp-modules/tasks/install-common.yml:48]: Remove tag

File: roles/ssp-modules/tasks/install-common.yml [hunk 7]
```diff
@@ -59,6 +69,8 @@
...
```
Comment (NicolasLiampotis) [inline roles/ssp-modules/tasks/install-common.yml:73]: Remove tag

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

**Review by nikosev — APPROVED** [review]
LGTM

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
LGTM

**Pull request merged**